components/trade-slip/TradeSlipItem.tsx

```diff
@@ -1,134 +0,0 @@
-import { isIndexedData } from "@zeitgeistpm/sdk-next";
```

Comment:
Could we keep the trade slip components for now? I think there are plans to have something similar so they might be useful